### ISSUE TITLE
Updates for DR8.

### ISSUE THREAD

**Issue by geordie666** [legacysurvey/legacysurvey#86]
A list of requests for DR8 (also potentially back-include in DR7):

- Add the full slate of status plots rather than just 1 or 2 (depth, seeing etc.)
- Add a link to the CCD issues page at http://www.ctio.noao.edu/noao/content/Status-DECam-CCDs
- Change "epoch" to "equinox" for J2000 in the RA/Dec or to ICRS without an equinox for Gaia (for all DRs).

**Comment by moustakas** [comment]
Shall we update the top-level title and description at 
http://legacysurvey.org from `The Legacy Surveys` to `DESI Legacy Imaging Surveys` to match the title of the overview paper?  May help prevent confusion...

**Comment by geordie666** [comment]
Note that the matching radius for external catalogs was increased to 1.5" for DR8.

**Comment by moustakas** [comment]
Some additional data model changes to the DR8 [survey-ccds](http://legacysurvey.org/dr7/files/#survey-ccds-dr7-fits-gz) and [annotated-ccds](http://legacysurvey.org/dr7/files/#ccds-annotated-dr7-fits-gz) files:

New columns:
* `plver`: Community Processing (CP) version number
* `procdate`: CP processing date
* `plprocid`: unique CP processing hash
* `airmass`: airmass of observation (measured at the telescope bore-sight)
* `ccdskysb`: sky surface brightness (AB mag/arcsec2)
* `ccdnastrom`: number of stars (after sigma-clipping) used to compute astrometric correction
* `ccdnphotom`: number of Gaia+PS1 stars detected with signal-to-noise ratio greater than five.

Removed columns:
* `ccdnmatch`

Some other notes:
* For `fwhm` we say "use `seeing` instead, but `seeing` doesn't exist.  This should state that this is the FWHM (in pixels) measured by the CP.

**Comment by moustakas** [comment]
For the documentation: the only known situation where `-photom` files would be missing (but `-annotated` and `-survey` files present) are if PSFex fails.  This is OK. See discussion in https://github.com/legacysurvey/legacypipe/issues/349.

**Comment by moustakas** [comment]
More updates:
* Output Tractor catalogs now contain only `grz` (https://github.com/legacysurvey/legacypipe/pull/356), reducing the number of columns from 204 to 160!!
* New WISE depth columns `psfdepth_w[1-4]` (https://github.com/legacysurvey/legacypipe/pull/358).
* New `maskbits` column with definitions that we should provide (https://github.com/legacysurvey/legacypipe/issues/279#issuecomment-482819549).

**Comment by moustakas** [comment]
New morphological type `DUP`: Gaia stars that are also in the LSLGA.  No point source is fitted (i.e., the fluxes and inverse variances are all zero), but all Gaia catalog entries are populated.

**Comment by geordie666** [comment]
`BRIGHTBLOB` gets set based on *initial* position (source detection integer pixel peak), while `MASKBITS` gets set after fitting has finished (i.e., on the final brick pixel position).

**Comment by moustakas** [comment]
The implication of [this comment](https://github.com/legacysurvey/legacysurvey/issues/86#issuecomment-487690059) is that a source can have the `MASKBITS` `BRIGHT` bit set but *not* `BRIGHTBLOB`, and vice versa.

**Comment by moustakas** [comment]
@geordie666 Could we add this little code tidbit somewhere on the DR8 website for converting `plprocid` to a date?  I've found myself having to dig this out of email several times, which probably means it's a useful thing to have on a webpage!  Thanks.

```bash
# Convert procid to date.
set procid = `echo $1 | tr '[:lower:]' '[:upper:]'`
set unixtime = `echo "ibase=16; $procid * A" | bc`
date -d @$unixtime '+%Y%m%dT%H%M%S’
```

For example:
```
% procid2date 91adb7f
20180528T163830
```

**Comment by dstndstn** [comment]
In python,

```
import datetime
datetime.datetime.fromtimestamp(0x8dfc6fa * 10)
# Produces: datetime.datetime(2017, 3, 6, 15, 25, 40)
```

**Comment by djschlegel** [comment]
The "Galactic Extinction" section of the docs don't give the coefficients used for the WISE filters and how they were derived.  Please include the text for this from:
https://desi.lbl.gov/trac/wiki/DecamLegacy/DR1Catalogs#Galacticextinctioncoefficients

**Comment by weaverba137** [comment]
Closed by #89.
